docs/build/using-local-fluence-network.md

```diff
@@ -0,0 +1,68 @@
+# Using local Fluence Network
```

Comment:
In order to deploy a deal to the local Fluence network, I need private key of the wallet, right?

How do I get chain's private keys in this setup?

Gotta explain how to use `docker logs` to obtain it, I guess...

Reply:
If you read further you will see explanation about that. The better way is to not require people to read docker logs, but to have a default key and if they want some other available key - I am printing that when you run `fluence chain info`